app/assets/stylesheets/navbar.scss

```diff
@@ -0,0 +1,49 @@
+.navbar-custom {
+    background-color: #169cb7;
+}
+
+.navbar-custom ul {
+  width: 100%
+}
+
+.navbar-nav li a {
+  color: black;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 20px;
+}
+
+.navbar-nav li a:hover {
+  background-color: #169cb7;
+}
+
+.navbar-nav li a:focus {
+    background-color: #grey;
+}
+
+// .navbar-custom .navbar-nav > li > a {
+//     color:#fff;
+// }
+//
+// .navbar-custom .navbar-nav > .active > a {
+//     color: #ffffff;
+//     background-color:transparent;
+// }
+
+// .navbar-custom .navbar-nav > li > a:hover,
+// .navbar-custom .navbar-nav > li > a:focus,
+// .navbar-custom .navbar-nav > .active > a:hover,
+// .navbar-custom .navbar-nav > .active > a:focus,
+// .navbar-custom .navbar-nav > .open >a {
+//     text-decoration: none;
+//     background-color: #33aa33;
+// }
+//
+// .navbar-custom .navbar-brand {
+//     color:#eeeeee;
+// }
+// .navbar-custom .navbar-toggle {
+//     background-color:#eeeeee;
+// }
+// .navbar-custom .icon-bar {
+//     background-color:#33aa33;
+// }
```

Comment:
just for future reference: We generally don't leave commented out code in 'live'  code